src/region/RegionCreator.scss

```diff
@@ -1,4 +1,4 @@
 .ba-RegionCreator.is-active {
-    // prettier-ignore
-    cursor: url('data:image/svg+xml;utf8,<svg version="1.1" height="32" width="32" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><g transform="translate(8 5)" fill-rule="nonzero"><polygon points="15 10 8.99 10 8.99 4 6.01 4 6.01 10 0 10 0 13 6.01 13 6.01 19 8.99 19 8.99 13 15 13" fill="%23fff"/><polygon points="13.99 11.01 7.99 11.01 7.99 5 7.01 5 7.01 11.01 1.01 11.01 1.01 11.99 7.01 11.99 7.01 18 7.99 18 7.99 11.99 13.99 11.99" fill="%23231F1F"/><path d="m19 0h-8v8h8v-8zm-3 3v2h-2v-2h2z" fill="%23fff"/><path d="m18 1h-6v6h6v-6zm-1 1v4h-4v-4h4z" fill="%23231F1F"/></g></g></svg>'), crosshair;
+    cursor: url('../icons/region_cursor_32.png') 16 16, crosshair; /* Legacy */
```

Comment:
Are these the only external images we use? If so, would it be simpler to base64 encode them in a stylesheet to avoid needing additional loaders?

Reply:
> Are these the only external images we use? If so, would it be simpler to base64 encode them in a stylesheet to avoid needing additional loaders?

Good idea, lemme try